pkg/ccl/importccl/exportcsv_test.go

```diff
@@ -360,6 +367,24 @@ func TestBasicParquetTypes(t *testing.T) {
 
 	err = validateParquetFile(t, paths[0], truth)
 	require.NoError(t, err)
+
+	// Test EXPORT PARQUET WITH NULL VALUES
+	sqlDB.Exec(t, `INSERT INTO foo VALUES (4, 'Alex', 3, 14.3, NULL), (5, 'Bobby', 2, NULL,false),
+(6, NULL, NULL, NULL, NULL)`)
+
+	sqlDB.Exec(t, `EXPORT INTO PARQUET 'nodelocal://0/null_parquet' FROM SELECT *
+		FROM foo ORDER BY x ASC LIMIT 2`)
```

Comment:
Are we getting much out of including the `ORDER BY ...` clauses here?

If we add another case to this test in the next PR, let's convert it to a table-driven test.

Reply:
Could you clarify what you mean by "table-driven test?"

Reply:
Happy to. "Table driven tests" is the name the go community has given to structuring your tests such that you can iterate over a slice of structs and share test code between test cases. For example: https://github.com/cockroachdb/cockroach/blob/master/pkg/ccl/importccl/exportcsv_test.go#L391 and https://github.com/cockroachdb/cockroach/blob/master/pkg/ccl/importccl/import_stmt_test.go#L126

Reply:
done.

Reply:
I like the order by clauses, as I can explicitly test that the parquet exporter is ordering rows correctly